integreat_cms/static/src/js/unsaved-warning.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * This file contains a function to warn the user when they leave a content without saving
+ */
+
+ var confirmed = false;
+ var edited = false;
```

Comment:
```suggestion
let confirmed = false;
let edited = false;
```
As far as I know  `let` is preferred over `var` because of the scoping.